third_party/re2/util/mutex.h

```diff
@@ -93,8 +93,8 @@ void Mutex::ReaderUnlock() { ReleaseSRWLockShared(&mutex_); }
 
 #define SAFE_PTHREAD(fncall)    \
   do {                          \
-    if ((fncall) != 0) abort(); \
-  } while (0)
+    if ((fncall) != 0) throw std::runtime_error("RE2 pthread failure"); \
+  } while (0);
```

Comment:
Do we want the trailing semicolon here?

```suggestion
  } while (0)
```